stratum/hal/lib/barefoot/bf_pipeline_utils.cc

```diff
@@ -72,76 +30,6 @@ std::string Uint32ToLeByteStream(uint32 val) {
 }
 }  // namespace
 
-::util::Status ExtractBfPipelineConfig(
```

Comment:
@bocon13 After removing format 2, this function collapsed to basically
```c++
  bf_config->Clear();
  if (bf_config->ParseFromString(config.p4_device_config())) {
    return util::OkStatus();
  } else {
   RETURN_ERROR(ERR_INVALID_PARAM) << "Unknown format for p4_device_config";
  }
```

and I removed it and its tests completely. But maybe there is value in having it as a standalone function?

Reply:
I agree that this method is perhaps trivial now, but if we decide to support other formats in the future (e.g. the message format directly from the SDE's bfrt.proto) then this might be useful.